packages/contracts-bedrock/test/libraries/SafeCall.t.sol

```diff
@@ -173,7 +177,10 @@ contract SafeCall_Test is Test {
             // Because forge coverage always runs with the optimizer disabled,
             // if forge coverage is run before testing this with forge test or forge snapshot, forge clean should be
             // run first so that it recompiles the contracts using the foundry.toml optimizer settings.
-            if (vm.isContext(VmSafe.ForgeContext.Coverage)) {
+            if (
+                vm.isContext(VmSafe.ForgeContext.Coverage)
+                    || LibString.eq(vm.envOr("FOUNDRY_PROFILE", string("default")), "lite")
+            ) {
```

Comment:
Perhaps worth pulling this out into a utility function? I don't feel strongly here, 3 instances isn't too many

Reply:
Eh for now I'll leave it as-is but if this gets worse we can pull it out